pkg/project/registry/projectrequest/delegated/delegated.go

```diff
@@ -96,7 +94,7 @@ func (r *REST) Create(ctx kapi.Context, obj runtime.Object) (runtime.Object, err
 		}
 	}
 
-	list, err := r.openshiftClient.TemplateConfigs(r.openshiftNamespace).Create(template)
```

Comment:
I'm confused... this is submitting the template to the server to process? It doesn't actually persist anything, right? Does it matter what namespace we submit to as long as it exists?